app/StateManager.js

```diff
@@ -251,6 +251,13 @@ var StateManager = Em.StateManager.extend(
         }),
 
         seat: Em.State.create({}),
+        obssh: Em.State.create({

```

Comment:
@YarikMamykin just a suggestion to keep consistency: either add here an empty line or remove in other places.